mne_sandbox/connectivity/cfc.py

```diff
@@ -0,0 +1,564 @@
+import numpy as np
+from mne.filter import band_pass_filter
+from mne.utils import _time_mask
+from mne.time_frequency import cwt_morlet
+from mne.preprocessing import peak_finder
+from mne.utils import ProgressBar
+import mne
+import warnings
+
+
+# Supported PAC functions
+_pac_funcs = ['plv', 'glm', 'mi_tort', 'mi_canolty', 'ozkurt', 'otc']
+# Calculate the phase of the amplitude signal for these PAC funcs
+_hi_phase_funcs = ['plv']
+
+
+def phase_amplitude_coupling(inst, f_phase, f_amp, ixs, pac_func='ozkurt',
+                             ev=None, tmin=None, tmax=None, n_cycles=None,
+                             scale_amp_func=None,  return_data=False,
+                             concat_epochs=False, n_jobs=1, verbose=None):
+    """ Compute phase-amplitude coupling between pairs of signals using pacpy.
+
+    Parameters
+    ----------
+    inst : an instance of Raw or Epochs
+        The data used to calculate PAC
+    f_phase : array, dtype float, shape (2,)
+        The frequency range to use for low-frequency phase carrier.
+    f_amp : array, dtype float, shape (2,)
+        The frequency range to use for high-frequency amplitude modulation.
+    ixs : array-like, shape (n_pairs x 2)
+        The indices for low/high frequency channels. PAC will be estimated
+        between n_pairs of channels. Indices correspond to rows of `data`.
+    pac_func : string, list of strings
```

Comment:
I think the convention is to say
`pac_func : {'plv', 'glm', 'mi_canolty', 'mi_tort', 'ozkurt'} | list of strings`